device:
  mirte:
    type: breadboard
    mcu: pico
    max_frequency: 10
distance:
  left:
    name: left
    device: mirte
    pins:
      trigger: 7
      echo: 6
encoder:
  left:
    name: left
    device: mirte
    pins:
      pin: 14             # pin needs interrupt support
    ticks_per_wheel: 20

